From 2c339bc164d47e9d08a5ee33e7163c18ae2c85aa Mon Sep 17 00:00:00 2001
From: Pete Beardmore <pete.beardmore@msn.com>
Date: Wed, 6 Sep 2017 15:53:16 +0100
Subject: remove redundant attribute tests

---
 quodlibet/quodlibet/plugins/gui.py | 28 +++++++++++++---------------
 1 file changed, 13 insertions(+), 15 deletions(-)

diff --git a/quodlibet/quodlibet/plugins/gui.py b/quodlibet/quodlibet/plugins/gui.py
index 23e261c..331ea8b 100644
--- a/quodlibet/quodlibet/plugins/gui.py
+++ b/quodlibet/quodlibet/plugins/gui.py
@@ -50,23 +50,21 @@ class UserInterfacePluginHandler(PluginHandler):
         self.__plugins[plugin.cls] = pl_obj = plugin.get_instance()
 
         # sidebars
-        if hasattr(pl_obj, 'create_sidebar'):
-            sidebar = pl_obj.create_sidebar()
-            app.window.hide_side_book()
-            if sidebar:
-                print_d("Enabling sidebar for %s" % plugin.cls)
-                self.__sidebars[plugin] = app.window.add_sidebar(
-                    sidebar, name=plugin.name)
-                sidebar.show_all()
+        sidebar = pl_obj.create_sidebar()
+        app.window.hide_side_book()
+        if sidebar:
+            print_d("Enabling sidebar for %s" % plugin.cls)
+            self.__sidebars[plugin] = app.window.add_sidebar(
+                sidebar, name=plugin.name)
+            sidebar.show_all()
 
         # widgetbars
-        if hasattr(pl_obj, 'create_widgetbar'):
-            widgetbar = pl_obj.create_widgetbar()
-            if widgetbar:
-                print_d("Enabling widgetbar for %s" % plugin.cls)
-                self.__widgetbars[plugin] = widgetbar
-                app.window.add_widgetbar(widgetbar)
-                widgetbar.show_all()
+        widgetbar = pl_obj.create_widgetbar()
+        if widgetbar:
+            print_d("Enabling widgetbar for %s" % plugin.cls)
+            self.__widgetbars[plugin] = widgetbar
+            app.window.add_widgetbar(widgetbar)
+            widgetbar.show_all()
 
     def plugin_disable(self, plugin):
 
-- 
1.8.4

