From 14df932628474578da42d119cf2bef8e6ea95d32 Mon Sep 17 00:00:00 2001
From: Pete Beardmore <pete.beardmore@msn.com>
Date: Fri, 25 May 2018 11:26:48 +0100
Subject: widgetbars, fix incorrect parameter name in multipaned

---
 quodlibet/quodlibet/qltk/paned.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/quodlibet/quodlibet/qltk/paned.py b/quodlibet/quodlibet/qltk/paned.py
index 8ef412754..c81d16087 100644
--- a/quodlibet/quodlibet/qltk/paned.py
+++ b/quodlibet/quodlibet/qltk/paned.py
@@ -526,12 +526,12 @@ class MultiPaned(object):
             # the last widget completes the last paned
 
             pack_expand = packing[min(i, len(packing) - 1)][0]
-            pack_fill = packing[min(i, len(packing) - 1)][1]
+            pack_shrink = packing[min(i, len(packing) - 1)][1]
 
             if widget is widgets[-1]:
-                curr_paned.pack2(widget, pack_expand, pack_fill)
+                curr_paned.pack2(widget, pack_expand, pack_shrink)
                 break
-            curr_paned.pack1(widget, pack_expand, pack_fill)
+            curr_paned.pack1(widget, pack_expand, pack_shrink)
 
             # the second last widget ends the nesting
             if widget is widgets[-2]:
-- 
2.14.1

