From 282548da2bf75858b40a48199d5ba791ae661413 Mon Sep 17 00:00:00 2001
From: Pete Beardmore <pete.beardmore@msn.com>
Date: Mon, 23 Sep 2013 17:57:11 +0100
Subject: autoconf, fall back to default datadir install location

---
 configure.ac | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/configure.ac b/configure.ac
index b3becac..5c0676d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -27,16 +27,18 @@ AM_MISSING_PROG(GIT2CL, git2cl, $missing_dir)
 # Check whether --with-lispdir was given.
 if test "${with_lispdir+set}" = set; then :
 else
-   mydir=$(dirname $0)
-   with_lispdir=$(EMACS_PROG=$EMACS $SH_PROG $mydir/compute-lispdir.sh)
+  my_lispdir=$(EMACS_PROG=$EMACS $SH_PROG $(dirname $0)/compute-lispdir.sh)
+  if test "${my_lispdir+set}" = set; then :
+    with_lispdir=$my_lispdir
+    echo "'compute-lispdir.sh' lispdir install directory override: '$with_lispdir'"
+  fi
 fi
 
 ##
 ## Find out where to install the debugger emacs lisp files
 ##
 AM_PATH_LISPDIR
+AM_CONDITIONAL(INSTALL_EMACS_LISP, test "x$lispdir" != "x")
 
 AC_CONFIG_FILES([Makefile test/Makefile])
 AC_OUTPUT
-echo
-echo "Install lisp directory set to $with_lispdir"
-- 
1.8.2

