From 111d22e68d0ee7b05b895191a2b18f52f7fb9a5d Mon Sep 17 00:00:00 2001
From: Pete Beardmore <pete.beardmore@msn.com>
Date: Sun, 1 Sep 2013 11:58:17 +0100
Subject: fix: don't access profile properties prematurely

---
 src/gamelogic/ui/ui_main.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/gamelogic/ui/ui_main.c b/src/gamelogic/ui/ui_main.c
index ae5d428..4f02527 100644
--- a/src/gamelogic/ui/ui_main.c
+++ b/src/gamelogic/ui/ui_main.c
@@ -3880,6 +3880,9 @@ static void UI_RunMenuScript( char **args )
 		}
 		else if ( Q_stricmp( name, "applyProfile" ) == 0 )
 		{
+			if ( uiInfo.profileList[ uiInfo.profileIndex ].name == NULL )
+				// no default profile yet
+				return;
 			Q_strncpyz( cl_profile.string, uiInfo.profileList[ uiInfo.profileIndex ].name, sizeof( cl_profile.string ) );
 			Q_CleanStr( cl_profile.string );
 			Q_CleanDirName( cl_profile.string );
@@ -3892,6 +3895,9 @@ static void UI_RunMenuScript( char **args )
 		{
 			fileHandle_t    f;
 
+			if ( uiInfo.profileList[ uiInfo.profileIndex ].name == NULL )
+				// no default profile yet
+				return;
 			Q_strncpyz( cl_defaultProfile.string, uiInfo.profileList[ uiInfo.profileIndex ].name, sizeof( cl_profile.string ) );
 			Q_CleanStr( cl_defaultProfile.string );
 			Q_CleanDirName( cl_defaultProfile.string );
-- 
1.8.2

