From ff6c86f65dc64378265460b3315c910a47ac06ec Mon Sep 17 00:00:00 2001
From: Pete Beardmore <pete.beardmore@msn.com>
Date: Sun, 1 Sep 2013 21:01:03 +0100
Subject: fix: 'nix, don't make assumptions on display name

---
 src/engine/sys/sys_unix.c | 22 +++++++++++++++++++---
 1 file changed, 19 insertions(+), 3 deletions(-)

diff --git a/src/engine/sys/sys_unix.c b/src/engine/sys/sys_unix.c
index 4fc7356..4e0734c 100644
--- a/src/engine/sys/sys_unix.c
+++ b/src/engine/sys/sys_unix.c
@@ -1136,11 +1136,27 @@ Sys_IsNumLockDown
 qboolean Sys_IsNumLockDown( void )
 {
 #if !defined(MACOS_X) && !defined(DEDICATED) && !defined(BUILD_TTY_CLIENT)
-	Display        *dpy = XOpenDisplay(":0");
+	const char     *denv = getenv( "DISPLAY" );
+	Display        *dpy;
 	XKeyboardState x;
 
-	XGetKeyboardControl(dpy, &x);
-	XCloseDisplay(dpy);
+	if ( denv != NULL && strlen( denv ) > 0 )
+		dpy = XOpenDisplay(denv);
+	else
+		dpy = XOpenDisplay(":0");
+
+	if ( dpy == 0 )
+	{
+		Com_Printf( _("ERROR: cannot determine numlock state as we couldn't\n\
+		                grab your non-standard (e.g. not ':0') X display.\n\
+		                ensure the 'DISPLAY' environment variable is set.\n") );
+		return qtrue;
+	}
+	else
+	{
+		XGetKeyboardControl(dpy, &x);
+		XCloseDisplay(dpy);
+	}
 
 	return (x.led_mask & 2) ? qtrue : qfalse;
 #else
-- 
1.8.2

